homework/Vasili_Zasinets/Shape/src/test/java/com/tutrit/java/quickstart/shape/CircleTest.java

```diff
@@ -0,0 +1,44 @@
+package com.tutrit.java.quickstart.shape;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CircleTest {
+    Circle circleTest = new Circle("Circle", 6);
+    Circle circleTestException = new Circle("CircleTest", -6);
```

Comment:
you'd better use fabric method to create such test data.

Reply:
Fixed. used the Factory pattern in tests.